cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/default-service/AwsDefaults.service.tsx

```diff
@@ -14,7 +15,46 @@ export class AwsDefaults {
         return first<AWSKeyPair>(keyPairs);
     };
 
-    static setDefaultNodeType = (nodeTypeList: string[], nodeProfile: string) => {
+    static setDefaultNodeType = async (nodeProfile: string) => {
+        const nodeTypeList: string[] = await AwsService.getAwsNodeTypes();
+        return AwsDefaults.defaultNodeTypeStategy(nodeTypeList, nodeProfile);
+    };
+
+    static setDefaultVpc = async (nodeProfile: string) => {
```

Comment:
I understand why you named this 'setDefaultVpc' since the availability zones are within the VPC object. However I think we should name this 'setDefaultAvailabilityZones' since we are using this method to set that list of az's

Reply:
Yes sir. Renamed.